src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -1460,12 +1277,12 @@ def test_vector_field_deref():
     for dist_mem in [True, False]:
         psy = PSyFactory(TEST_API,
                          distributed_memory=dist_mem).create(invoke_info)
-        generated_code = psy.gen
-        assert str(generated_code).find("SUBROUTINE invoke_0_testkern_chi_"
-                                        "type(f1, box_chi, f2)") != -1
-        assert str(generated_code).find("TYPE(field_type), intent(inout)"
-                                        " :: f1, box_chi(3)") != -1
-        assert "TYPE(field_type), intent(in) :: f2" in str(generated_code)
+        generated_code = str(psy.gen)
+        assert ("SUBROUTINE invoke_0_testkern_chi_type(f1, box_chi, f2)" in
+                generated_code)
+        assert ("TYPE(field_type), intent(inout) :: f1, box_chi(3)" in \
```

Comment:
pycodestyle error:


tests/dynamo0p3_test.py:1283:72: E502 the backslash is redundant between brackets

Reply:
Done.